gpkit/nomials/nomial_core.py

```diff
@@ -61,7 +62,7 @@ def latex(self, excluded=None):
         if "units" in excluded:
             return " + ".join(sorted(mstrs))
 
-        units = unitstr(self.units, r"\mathrm{~\left[ %s \right]}", "L~")
+        units = self.unitstr(r"\mathrm{~\left[ %s \right]}", "L~")
         units_tf = units.replace("frac", "tfrac").replace(r"\cdot", r"\cdot ")
```

Comment:
I'm referring to the line `units_tf = units.replace("frac", "tfrac").replace(r"\cdot", r"\cdot ")`. Wondering why that is living in `Nomial`.